.github/scripts/docker_push_script.sh

```diff
@@ -0,0 +1,28 @@
+# Names of the images created for each architecture
```

Comment:
Add `#!/bin/bash` to all scripts. Also ensure your `VERSION` variable is valid (not null/empty) before tagging and pushing.

Reply:
Done. If version is not set, script will echo that the docker image wasn't pushed.